### PR TITLE
V3

### PR DESCRIPTION
**Pull request by kishanm-wtbx**
For review: 
- Add error handler for API calls to avoid undefined variable error.
- Change to load embedded code script

Ticket Link: [Client Ticket Link](https://www.websitetoolbox.com/tool/ticket/view/51030/36zKhmVI4uAtv9YyWaNw)

Tester Name: -

### CODE REVIEW

File: .craftplugin [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+{"pluginName":"Website Toolbox Community","pluginDescription":"Your CraftCMS community doesn’t need to be basic. Website Toolbox is a cloud-based community plugin that allows you to easily add a powerful, intuitive, and maintenance-free community to CraftCMS without slowing it down. No database, servers, or coding required","pluginVersion":"1.0.0","pluginAuthorName":"Website Toolbox","pluginVendorName":"Website Toolbox","pluginAuthorUrl":"http://websitetoolbox.com","pluginAuthorGithub":"","codeComments":"yes","pluginComponents":["services","settings"],"consolecommandName":"","controllerName":"","cpsectionName":"","elementName":"","fieldName":"","modelName":"","purchasableName":"","recordName":"","serviceName":"","taskName":"","utilityName":"","widgetName":"","apiVersion":"api_version_3_0"}
```
Comment (webtoolbox) [inline .craftplugin]: Use the description that was approved for version 4.

File: README.md [hunk 2]
```diff
@@ -15,4 +19,10 @@ To install Website Toolbox Community, follow these steps:
 
 You can also install Website Toolbox Community via the **Plugin Store** in the Craft CP.
 
+Website Toolbox Community works on Craft 3.x.
+
+## Documentation
+
+Click here -> [Website Toolbox Community Documentation](https://websitetoolbox.com/plugins/websitetoolboxforum/documentation)
```
Comment (webtoolbox) [inline README.md]: This link does not exist! Feedback was already given on that previously. Remove it.

File: composer.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
     "name": "websitetoolbox/websitetoolboxcommunity",
-    "description": "The Easiest CraftCMS Community Plugin. Effortlessly Build a Beautiful Community Forum with Zero Coding, Seamless Embedding, Instant Setup, and Exceptional Support.",
+    "description": "Your CraftCMS community doesn’t need to be basic. Website Toolbox is a cloud-based community plugin that allows you to easily add a powerful, intuitive, and maintenance-free community to CraftCMS without slowing it down. No database, servers, or coding required",
```
Comment (webtoolbox) [inline composer.json]: Use the previously approved description.

**Review by webtoolbox — CHANGES_REQUESTED** [review]

**Review by webtoolbox — DISMISSED** [review]

**Review by masihur — DISMISSED** [review]
In Some places you are using community and on some places you are using forum. Please check it properly.

**Comment by kishanm-wtbx** [comment]
It is because of craftcms support team suggest us to keep the plugin handle name same as we have in earlier version. We can change plugin name but cannot change plugin handle name. If we change the handle name then we have to launch the plugin as completely new. That is the reason to code contain forum at some place and community at rest place.

**Review by webtoolbox — DISMISSED** [review]

**Comment by kishanm-wtbx** [comment]
Sandeep Sir,

As per the craftcms support team, we need to keep separate branches for version 3 and version 4. So merge request is not require from my side. You can link/upload the v3 brach to craftcms store. It may be done from craftcms store account.

**Comment by kishanm-wtbx** [comment]
The v3 branch itself a branch for craftcms version 3 plugin. It has been reviewed by Sandeep sir and Masihur sir. The pull request no need to merge with any other branch. So, I close it.

**Pull request closed without merging**